creusot/tests/should_fail/bug/222.coma

```diff
@@ -154,9 +154,9 @@ module M_222__uses_invariant
     | & _5 : borrowed (Option'0.t_Option t) = any_l () ]
      [ return' (result:())-> {[@expl:postcondition] [%#s2222] invariant'0 x.final} (! return' {result}) ] 
 end
-module M_222__qyi2574917115831112192
+module M_222__qyi2574917115831112192 [#"../222.rs" 19 0 19 13] (* <() as A> *)
```

Comment:
I wonder if Why3 has a feature that lets us remap the names in the ID, it would definitely be preferable to see "<() as A>" to `M_222__qyi2574917115831112192`